src/server_manager/ui_components/outline-manual-server-entry.html

```diff
@@ -190,18 +216,39 @@
                   <li>[[localize('gcp-firewall-create-2')]]</li>
                   <li>[[localize('gcp-firewall-create-3')]]</li>
                   <li>[[localize('gcp-firewall-create-4')]]</li>
-                  <li>[[localize('gcp-firewall-create-5')]]</li>
+                  <li>[[localize('gcp-click-create')]]</li>
                 </ol>
               </outline-cloud-instructions-view>
             </div>
+          </iron-collapse>
+        </div>
+        <div class="section" hidden$="[[!isCloudProviderGcp]]">
+          <div class="section-header">
+            <!-- TODO(alalama): localize numbers  -->
+            <span class="stepcircle">3</span>
+            <!-- TODOBEFOREMERGE i18n -->
+            <div class="instructions">Create your server</div>
+            <div class="drop-down" on-tap="_toggleGcpCreateServerDropDown">
+              <img src="images/gcp-logo.svg" />
+              <span>[[localize('manual-server-instructions')]]</span>
+              <iron-icon id="gcpCreateServerDropDownIcon" icon="arrow-drop-down"></iron-icon>
+            </div>
+          </div>
+          <iron-collapse id="gcpCreateServerDropDown">
             <div class="section-content-instructions">
-              <outline-cloud-instructions-view title="[[localize('manual-server-assign-firewall')]]" thumbnail-path="images/gcp-thumbnail-2.png" image-path="images/gcp-screenshot-2.png"" localize="[[localize]]">
+              <!-- TODOBEFOREMERGE i18n -->
+              <!-- TODOBEFOREMERGE images -->
+              <outline-cloud-instructions-view title="Create a VM Instance" thumbnail-path="images/fine.png" image-path="images/greatwave.jpg" localize="[[localize]]">
                 <ol>
-                  <li inner-h-t-m-l="[[localize('gcp-firewall-assign-0', 'openLink', '<a href=https://console.cloud.google.com/compute/instances>', 'closeLink', '<iron-icon icon=open-in-new></iron-icon></a>')]]"></li>
-                  <li>[[localize('gcp-firewall-assign-1')]]</li>
-                  <li>[[localize('gcp-firewall-assign-2')]]</li>
-                  <li>[[localize('gcp-firewall-assign-3')]]</li>
-                  <li>[[localize('gcp-firewall-assign-4')]]</li>
+                  <li inner-h-t-m-l="[[localize('gcp-create-vm', 'openLink', '<a href=https://console.cloud.google.com/compute/instancesAdd>', 'closeLink', '<iron-icon icon=open-in-new></iron-icon></a>')]]"></li>
+                  <li>Type "outline-server" in the "Name" field</li>
+                  <li>Select a region close to where the server's users will be under "Region"</li>
+                  <li>Select "f1-micro" under "Machine type"</li>
+                  <li>Click "Change" under "Boot disk"</li>
+                  <li>Choose "CoreOS" under Operating System, and "CoreOS stable" under Version</li>
```

Comment:
How did you choose CoreOS vs other images such as Debian?

Reply:
CoreOS is small and comes with Docker pre-installed

Reply:
That it comes with Docker installed is nice, but we reinstall it anyway, no?

I don't know how minimal CoreOS is. My worry about minimal OSes is that they are annoying to debug and work with. (e.g. shadowbox: no tools, no bash, etc...) A more complete user-friendly OS makes things a bit easier. Disk space is not a significant issue. Does CoreOS have bash and curl? We need that for the install script.

Reply:
Done in 25ee445.  We now just keep with the default of Debian